modules/installation/pages/setup-wizard.adoc

```diff
@@ -1,29 +1,36 @@
-[[ref.webui.admin.wizard]]
+[[Setrup Wizard]]
```

Comment:
```suggestion
[[Setup Wizard]]
```
I guess we must clarify how these label should look. What will be the end result (file names of the HTML output).

Reply:
oh, it probably needs a hyphen, too.